src/Student5/Ex6/SentenceBuilder.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+ import { useRef, useState } from "react";
+ import "./SentenceBuilder.scss";
+ 
+ export const SentenceBuilder = (props) => {
+   let inputRef = useRef(null);
+   let [disable, setDisable] = useState();
+   let [counter, setCounter] = useState(0);
+   
+   const onHandleClick = () => {
+     inputRef.current.value = "jumps";
```

Comment:
The idea is to build a React component that lets the user build a sentence for the words that they type.  Hardcoding the word "jump" is not going to work here.